samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -88,6 +91,20 @@ static const struct bt_data ad[] = {
 
 int main(void)
 {
+	/* Zephyr Controller works best while Extended Advertising interval to be a multiple
+	 * of the ISO Interval minus 10 ms (max. advertising random delay). This is
+	 * required to place the AUX_ADV_IND PDUs in a non-overlapping interval with the
+	 * Broadcast ISO radio events.
+	 * For 10ms SDU interval a extended advertising interval of 60 - 10 = 50 is suitable
+	 */
```

Comment:
Can we add a comment:
```suggestion
	* Other controllers are able to generate conflict-free scheduling patterns without setting the advertising interval
	* in this way.
	*/
```

Reply:
Sure :)

Reply:
@cvinayak since we have samples that have host features not supported by the controller, there is already a general understanding/consensus that BT samples are not specific for any controller, not even the Zephyr controller. I'm unsure whether we should mention that specifically here. Coul also just be vague and say "some controllers"?

Reply:
+1 for mentioning that other controllers may not need this. We have a similar case for documenting scan window when CONFIG_BT_SCAN_AND_INITIATE_IN_PARALLEL is used.

Reply:
I was initially thinking it was fine to mention that the zephyr controller needs this but others don't. But I don't have a strong preference here